packages/sdk/src/sdk.ts

```diff
@@ -876,11 +875,45 @@ export class NxtpSdk {
   public async cancelExpired(cancelParams: CancelParams, chainId: number): Promise<providers.TransactionResponse> {
```

Comment:
Since this has nothing to do with expiries (i.e. user could, and should, cancel receiver-side transfers to signal to the router to unlock the sender side if there was a validation error, or some things changed). We should rename it to just `cancel`

Reply:
And do some basic validation on what chain is being cancelled and if it is expired (or use the tx service to run estimate gas and get the proper revert message)